data-prepper-api/src/test/java/com/amazon/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -333,6 +334,16 @@ public void testBuild_withTimeReceived() {
         assertThat(event.getMetadata().getTimeReceived(), is(equalTo(now)));
     }
 
+    @Test
+    public void testBuild_withMessageKey() {
+
+        String messageKey = UUID.randomUUID().toString();
```

Comment:
This variable should be renamed to `messageValue` since it holds the value, not the key.